app/graphql/albums/resolvers.js

```diff
@@ -0,0 +1,20 @@
+const config = require('../../../config'),
+  { url } = config.common.apiAlbums,
+  { getAlbumSources } = require('../../services/albums');
+
+const getAlbum = (_, params) =>
+  getAlbumSources(`${url}albums/${params.id}`).then(album => ({
+    id: album.id,
+    title: album.title,
+    artist: album.userId
+  }));
+
+module.exports = {
+  Query: {
+    album: getAlbum
+  },
+  Album: {
+    id: root => root.id,
+    photos: params => getAlbumSources(`${url}photos?albumId=${params.id}`)
```

Comment:
Think of the first argument of a resolver as the `root` or `parent`, naming it `params` may be confusing with second argument thats normally named `args` or something like `params`

Reply:
perfect!